packages/forma-36-website/src/pages/components/text-field/index.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: 'TextField'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/TextField'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-textfield--default'
+---
+
+The `TextField` component is a composition of the `FormLabel`, `TextInput`, `HelpText` and `ValidationMessage` components.
+It allows the user to input text or numbers and should be used in a Form context.
+
+## Best practices
+
+- Provide a labelText, name and id to ensure accessibility
+
+## Copy considerations
+
+Todo: Provide copy considerations
+
+## TextField variations
+
+The TextField component can be configured in a number of different ways. Here is a guide to when to use certain variations.
```

Comment:
The TextField component can be configured in a number of different ways. Use this guidance to determine which variation to use based on the context.